src/main/assembly/filter.properties

```diff
@@ -0,0 +1,2 @@
+product.name=WSO2 EI
+product.version=6.1.1
```

Comment:
Can we only define two filter tags here? Is there a way to mention `swift`, `iso20022` filters here?

Reply:
Will include those with connetor module implementations